seed/seed.go

```diff
@@ -160,26 +160,22 @@ type Seed interface {
 	Iter(f func(sn *Snap) error) error
 }
 
-// A SnapHandler can be used to perform any dedicated handling of seed
-// snaps and their digest computation while seed snap metadata loading
-// and verification is being performed.
-type SnapHandler interface {
-	// HandleAndDigestAssertedSnap should compute the digest of
-	// the given snap and perform any dedicated
-	// handling. essentialType is provided only for essential
-	// snaps.
-	// snapRev is provided by UC20+ seeds.
-	// deriveRev is provided by UC16/18 seeds, it can be used
-	// to get early access to the snap revision based on the digest.
-	// A different path can be returned if the snap has been copied
+// A ContainerHandler can be used to perform any dedicated handling of seed
+// snaps/containers and their digest computation while seed snap/container
```

Comment:
Should these be `snap/container` or `snap/component`?

Reply:
component indeed :slightly_smiling_face: